addons/web/static/src/js/views/form_widgets.js

```diff
@@ -888,6 +888,27 @@ var FieldSelection = common.AbstractField.extend(common.ReinitializeFieldMixin,
     }
 });
 
+var LabelSelection = common.AbstractField.extend(common.ReinitializeFieldMixin, {
```

Comment:
I would like to inherit selection field instead of abstractfield, current implementation make field readonly 
```javascript 
var LabelSelection = FieldSelection.extend({
    init: function(field_manager, node) {
        this._super(field_manager, node);
        this.classes = this.options && this.options.classes || {};
    },
    render_value: function() {
        this._super.apply(this, arguments);
        if (this.get("effective_readonly")) {
            var value = this.get('value'),
                bt_class = this.classes[value];
            if (bt_class){
                this.$el.html(_.str.sprintf("<span class='label label-%s'>%s<span>", bt_class, this.$el.html()))
            }
        }
    },
});